Const

redheadedrod opened the pull request:
This commit only contains my current attempt at the code. I understand it now but it is not working 100%
Submitting for your comment.

atlas0fd00m commented:
why a second PR?

On Tuesday, June 7, 2016 8:47:35 AM EDT redheadedrod wrote:
> This commit only contains my current attempt at the code. I understand it now but it is not working 100%
> Submitting for your comment.
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/atlas0fd00m/vivisect/pull/6
> 
> -- Commit Summary --
> 
>   * Code base re-merged and all previously removed code reintroduced.
>   * Added ldr tests, removed excess h8 test stuff no longer need, removed ida dump.
>   * Attempt at const update. Having issues. Question inlined in code
> 
> -- File Changes --
> 
>     M envi/archs/arm/__init__.py (18)
>     M envi/archs/arm/const.py (71)
>     M envi/archs/arm/disasm.py (17)
>     M envi/tests/test_arch_arm.py (1280)
> 
> -- Patch Links --
> 
> https://github.com/atlas0fd00m/vivisect/pull/6.patch
> https://github.com/atlas0fd00m/vivisect/pull/6.diff
> 
> ---
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/atlas0fd00m/vivisect/pull/6
-- 
S0ZNRkNaMlpLTkJHVVdTWUpJWUZTVjNNT1ZFVVFRVFdNRkxUS01DSkk1V0hLU0tJS0pZR0VWMlZPTkVVUVVUUExKSlVFNksyTEJERENZS1lKSldGVVEyQ09OTkZRV1RNTUpCVUU1UzJORkJHVVlSU0dGM1dHM0tXTjVORk9OTDJNRkxUUzVLSkpCSkhNU0tISVpWR0NSM01OUlNHMlZMSExGSlVFNlRFSzVIR1VXU1lKWjVGVTNTV09ORVVPVlJVTU5EWFE1VEJMQkpHUVpDSE5SM0dFMktDUEZORk9SVEtNRkRWTTZTSkpCSkc2V1NUSUpXRlVSM0VOUkVVT09MTkpGRFdZNUxER0pESEtZS1lLSTJVWTJLQ0tWUVVPVkxITU1aVUU0REROVldEQVNLSEdGV0dFU0NTUEpFVVFaRFFNUkRXT1ozQ0s1REdXWVROS1o1R0c2TFhNNVNERVZMSE1SRFZNNURESTQ0WFNXS1lKSllHRVNETE01UkVPT0xXTU1aRktaMlpLNTRIR1NLSE9SWUdFM0tSTTVSREVXTEhNTldVTU1EQks0NFhLV0tYUEJZR0lTRExNNU1WT05MTEpGRUdJM0NKSTVMSEtaQ0hLWjRVU1IyRk01UlRHVVRJTVJEVktaM0NHSk1XT1lLWFBCWldJVlpST0JSRzJSUlFNRkxUUzVLTU5GQVhJU0tJTEpWVkNTQ0NONVJXMlJUS01GNFRLNVRETlZSVVc9PT0=
-- 
353436383635373236353230363937333230364536463230364436313637363936333230373436463230363336463644373037353734363537323733324332303646364536433739323036443631363736393633363936313645373332453041
>

redheadedrod commented:
Was a different branch on my fork. 
I decided to do the unit test stuff on my 'unittest' branch
I did the changes to the const.py in a sub branch of that branch called 'const' so I could still work on the unit test stuff while not affecting the changes to const until that was solidified.
So my local fork is sort of like:
master -\
                Atlas_Arm_fixes-\
                                               unittest-\
                                                                const                                                    

I thought it should have added them to the same PR but apparently didn't. 

Still learning.. 

Date: Tue, 7 Jun 2016 12:32:05 -0700
From: notifications@github.com
To: vivisect@noreply.github.com
CC: redheadedrod@hotmail.com; author@noreply.github.com
Subject: Re: [atlas0fd00m/vivisect] Const (#6)

why a second PR?



On Tuesday, June 7, 2016 8:47:35 AM EDT redheadedrod wrote:

> This commit only contains my current attempt at the code. I understand it now but it is not working 100%

> Submitting for your comment.

> You can view, comment on, or merge this pull request online at:

> 

>   https://github.com/atlas0fd00m/vivisect/pull/6

> 

> -- Commit Summary --

> 

>   * Code base re-merged and all previously removed code reintroduced.

>   * Added ldr tests, removed excess h8 test stuff no longer need, removed ida dump.

>   * Attempt at const update. Having issues. Question inlined in code

> 

> -- File Changes --

> 

>     M envi/archs/arm/__init__.py (18)

>     M envi/archs/arm/const.py (71)

>     M envi/archs/arm/disasm.py (17)

>     M envi/tests/test_arch_arm.py (1280)

> 

> -- Patch Links --

> 

> https://github.com/atlas0fd00m/vivisect/pull/6.patch

> https://github.com/atlas0fd00m/vivisect/pull/6.diff

> 

> ---

> You are receiving this because you are subscribed to this thread.

> Reply to this email directly or view it on GitHub:

> https://github.com/atlas0fd00m/vivisect/pull/6

-- 

S0ZNRkNaMlpLTkJHVVdTWUpJWUZTVjNNT1ZFVVFRVFdNRkxUS01DSkk1V0hLU0tJS0pZR0VWMlZPTkVVUVVUUExKSlVFNksyTEJERENZS1lKSldGVVEyQ09OTkZRV1RNTUpCVUU1UzJORkJHVVlSU0dGM1dHM0tXTjVORk9OTDJNRkxUUzVLSkpCSkhNU0tISVpWR0NSM01OUlNHMlZMSExGSlVFNlRFSzVIR1VXU1lKWjVGVTNTV09ORVVPVlJVTU5EWFE1VEJMQkpHUVpDSE5SM0dFMktDUEZORk9SVEtNRkRWTTZTSkpCSkc2V1NUSUpXRlVSM0VOUkVVT09MTkpGRFdZNUxER0pESEtZS1lLSTJVWTJLQ0tWUVVPVkxITU1aVUU0REROVldEQVNLSEdGV0dFU0NTUEpFVVFaRFFNUkRXT1ozQ0s1REdXWVROS1o1R0c2TFhNNVNERVZMSE1SRFZNNURESTQ0WFNXS1lKSllHRVNETE01UkVPT0xXTU1aRktaMlpLNTRIR1NLSE9SWUdFM0tSTTVSREVXTEhNTldVTU1EQks0NFhLV0tYUEJZR0lTRExNNU1WT05MTEpGRUdJM0NKSTVMSEtaQ0hLWjRVU1IyRk01UlRHVVRJTVJEVktaM0NHSk1XT1lLWFBCWldJVlpST0JSRzJSUlFNRkxUUzVLTU5GQVhJU0tJTEpWVkNTQ0NONVJXMlJUS01GNFRLNVRETlZSVVc9PT0=

-- 

353436383635373236353230363937333230364536463230364436313637363936333230373436463230363336463644373037353734363537323733324332303646364536433739323036443631363736393633363936313645373332453041

> 

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
